hack/generate.sh

```diff
@@ -158,14 +158,20 @@ function generate_manifests() {
         kustomize build ${controller_crd_path} > ${controller_crd_path}/resources.yaml
         controller-gen object:headerFile=${hack_boilerplate} paths="./..."
         goimports -w  ${controller_path}
+        cd ..
```

Comment:
you can also replace function parenthesis to have regular brackets instead of curly brackets
it will make ``cd ./api`` an action relevant only to the function scope
```
function generate_manifests() (
    ...
)

Reply:
that comment is only an optional suggestion, not required ^

Reply:
Done